core/tauri/src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub mod command;
 /// The Tauri API endpoints.
 mod endpoints;
 mod error;
-mod event;
+pub mod event;
```

Comment:
don't export this whole mod. just do `pub use` for the types you need exported from the module like how it is done a few lines below.

Reply:
It seems that only Event and Eventhandler are needed directly.